### DISCUSSION TITLE
Outlook sync failed

### DISCUSSION THREAD

**Discussion by xathick** [shridhar-tl/jira-assistant#359] [Q&A]
I changed the password in Outlook as asked by company, but i'm not able to get it working again.
When ever I try to log in with "Click here to sign in with your microsoft account" redirects to microsoft (already signed in) and then fails.
Searching in the logs i found "AADSTS7000222: The provided client secret keys for app 'a8efd8fc-0657-490e-a622-7b2aaa4f4f46' are expired. Visit the Azure portal to create new keys for your app: https://aka.ms/NewClientSecret, or consider using certificate credentials for added security: https://aka.ms/certCreds. Trace ID: 76f6ad90-7185-48af-bc15-61756da00b00 Correlation ID: 2bf9ed39-d4b2-45f9-b44b-dbf2ef6b872c Timestamp: 2024-08-09 06:57:16Z"
But I never provided any key to jira assistant. The links redirects to a microsoft page where you can handle access and so on, but is admined by the organization, I have never been able to do it. 
Any tips or ideas?

Using Chromium on Windows 11.

**Comment by tonisc** [comment]
Hi
all my users have the same problem. Yesterday works fine, but today, we obtains the same error. We use all supports: Browser extension, applicaction installed through jira marketplace and WebService.

**Comment by xathick** [comment]
Could  then have been a MS change that we aren't aware of?

**Comment by tonisc** [comment]
I think that this is possible or I don't know if JiraAssistant has a "universal temporal token" or similar for use Api Graph. I don't understand how appairs the enterprise application "Jira Assistant" in my EntraID.

The problem is when the Enterprise Application Jira Assistant calls to api graph:
```
Authentication requirement Single-factor authentication
Status Failure
Continuous access evaluation	No
Sign-in error code 7000222
Failure reason The provided client secret keys for app '{identifier}' are expired. Visit the Azure portal to create new keys for your app: https://aka.ms/NewClientSecret, or consider using certificate credentials for added security: https://aka.ms/certCreds.
Additional Details Developer error - the app is attempting to sign in without the necessary or correct authentication parameters.
Application Jira Assistant
Application ID	a8efd8fc-0657-490e-a622-7b2aaa4f4f46
Resource	Microsoft Graph
Resource ID 00000003-0000-0000-c000-000000000000
Home tenant name	
Client app Browser
Client credential type None
Service principal ID	
Original transfer method None
Token Protection - Sign In Session	Unbound
Service principal name	
Resource service principal ID	46bde66a-9231-4bab-82ac-d51bcf897531
Unique token identifier	fo10iQ009UyuHYvXFe0XAA
Token issuer type	Microsoft Entra ID
Token issuer name	
Incoming token type None
Authentication Protocol	None
Latency 1293ms
Flagged for review No
User agent node
```

**Comment by shridhar-tl** [comment]
This issue has been resolved and you should be able to use outlook integration again.
